src/modules/core/components/zoom/zoom.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ZoomControl } from './zoom';
+
+const meta: Meta<typeof ZoomControl> = {
+  title: 'Core UI/Zoom',
```

Comment:
change to `Location UI/ZoomControl` and move `zoom` folder to the `modules/location/components` and rename folder name to `zoom-control` and files like `zoom-control.tsx`...